### PR TITLE
Basically the 2.x version ported to 3.x.

### PR DESCRIPTION
**Pull request by fgm**

### CODE REVIEW

File: src/GraphQL/Execution/Processor.php [hunk 1]
```diff
@@ -70,7 +70,7 @@ protected function getResolveFunction(AbstractField $field) {
   /**
    * {@inheritdoc}
    */
-  protected function resolveFieldValue(FieldInterface $field, $contextValue, array $fields, array $args) {
+  protected function resolveFieldValue(AbstractField $field, $contextValue, array $fields, array $args) {
```
Comment (fubhy) [inline src/GraphQL/Execution/Processor.php]: THis is a leftover I guess?

Comment (fgm) [reply src/GraphQL/Execution/Processor.php]: uh, oh, just noticed it too...

File: src/GraphQL/Field/Root/Entity/EntityQueryField.php [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+  /**
+   * The entity type handled by this field instance.
+   *
+   * @var string
+   */
+  protected $entityTypeId;
+
+  /**
+   * The entity_type.manager service.
```
Comment (fubhy) [inline src/GraphQL/Field/Root/Entity/EntityQueryField.php]: I usually write this out as a plain string ("The entity type manager service".

File: src/GraphQL/Field/Root/Entity/EntityQueryField.php [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+   *   The typed_data_manager service.
+   */
+  public function __construct(
+    EntityTypeInterface $entityType,
+    TypeInterface $outputType,
+    EntityTypeManagerInterface $entityTypeManager,
+    TypedDataManagerInterface $typedDataManager,
+    TypeResolverInterface $typeResolver) {
+    $this->entityTypeId = $entityType->id();
+    $this->entityTypeManager = $entityTypeManager;
```
Comment (fubhy) [inline src/GraphQL/Field/Root/Entity/EntityQueryField.php]: We cannot do this or we will have trouble serializing this field.

File: src/GraphQL/Execution/Processor.php (hunk 1 above)
Comment (fgm) [reply src/GraphQL/Execution/Processor.php]: No longer there. Funny GH doesn't notice it's disappeared.

File: src/GraphQL/Field/Root/Entity/EntityQueryField.php (hunk 2 above)
Comment (fgm) [reply src/GraphQL/Field/Root/Entity/EntityQueryField.php]: Removed anyway.

**Pull request merged**